docs/Tutorial.md

```diff
@@ -10,7 +10,7 @@ Admin-on-REST uses React. We'll use Facebook's [react-create-app](https://github
 npm install -g create-react-app
 create-react-app test-admin
 cd test-admin/
-npm install --save-dev admin-on-rest
+npm install --save-dev babel-runtime admin-on-rest
```

Comment:
The PR #9 aims the example folder, not the tutorial which use `create-react-app`